src/modules/wizard/mock-data.json

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
@titkov49 Do we need mocked data in the `modules/wizard`? It might be better, if it would be a part of documentation or `public` folder